lucet-wiggle/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "lucet-wiggle"
+version = "0.6.2-dev"
+homepage = "https://github.com/fastly/lucet"
+repository = "https://github.com/fastly/lucet"
+license = "Apache-2.0 WITH LLVM-exception"
+categories = ["wasm"]
+authors = ["Lucet team <lucet@fastly.com>"]
+edition = "2018"
+
+[lib]
+proc-macro = true
+
+[dependencies]
+lucet-wiggle-generate = { path = "generate", version = "0.6.2-dev" }
+wiggle-generate = { path = "../../wiggle/crates/generate" }
+witx = "0.8.3"
+syn = { version = "1.0", features = ["full"] }
+
+[dev-dependencies]
+wiggle-runtime = { path = "../../wiggle/crates/runtime" }
+wiggle-test = { path = "../../wiggle/crates/test" }
```

Comment:
Is this assuming that the `wasmtime` repository is adjacent to `lucet`?

Reply:
just being sloppy by developing it on my machine that way and making some minor changes in wiggle in parallel. wiggle hasnt landed in the wasmtime repo yet, but when it does we'll use it via crates.io, where it should end up as part of the normal wasmtime release process.

Reply:
Ah! Sorry to bother :slightly_smiling_face: